### ISSUE TITLE
compare train loss with larger models

### ISSUE THREAD

**Issue by sadransh** [jzhang38/TinyLlama#66]
is there any apple2apple comparison for this?

**Comment by RonanKMcGovern** [comment]
well the vocab is the same, so they could be compared if the same eval dataset was being used.

The eval dataset here though is likely a random sample of the training set and not matching the eval dataset used for Llama 2 models.

So I suppose you'd have to run a perplexity evaluation on both models to get an apples to apples comparison. You could do that using wikitext or other datasets on huggingface.

**Comment by jzhang38** [comment]
Yes, as @RonanKMcGovern pointed out, we use the val set of SlimPajama for validation. The dataset of Llama 1/2 has not been publicly released. Val loss here only serve the purpose of keeping track of the training progress. It is better to directly compare benchmark performance when comparing with larger models.
